virtcontainers/sandbox.go

```diff
@@ -1062,6 +1067,24 @@ func (s *Sandbox) stopVM() error {
 		s.Logger().Info("Not stopping VM")
 		return nil
 	}
+	pid := s.hypervisor.pid()
+
+	defer func() {
+		exited := false
+		for i := 0; i < waitRetry; i++ {
+			if running, _ := isProcRunning(pid); !running {
+				exited = true
+				break
+			}
+			syscall.Kill(pid, syscall.SIGKILL)
+			networkLogger().WithField("pid", pid).Warn("NOTE: hypervisor still running after stopVM")
+			time.Sleep(waitVMExitTime * time.Millisecond)
+		}
+		if !exited {
+			networkLogger().WithField("pid", pid).Info("NOTE: hypervisor not exited after 10s")
```

Comment:
`10s` can change depending of `waitRetry` and `waitVMExitTime`, could you calculate this value dynamically, something like `((waitRetry * waitVMExitTime) / time.Millisecond)`